### DISCUSSION TITLE
File Open keystrokes

### DISCUSSION THREAD

**Discussion by anjohnson** [eteran/nedit-ng#147] [Q&A]
The Getting Started help document has a section called [Shortcuts](https://github.com/anjohnson/nedit-ng/wiki/01#shortcuts) which describes navigating the File Open dialog box thusly:

> To open a file named `whole_earth.c`, type:
> 
>     Ctrl+O who Return
> 
> (how much of the filename you need to type depends on the other files in the directory).

The keystrokes described no longer work exactly as they used to, I have to press Down-arrow before Return to select the specific file from the drop-down list where the Qt dialog presents the matching filename choices.

Should this discrepancy be fixed in the code, or in the Help documentation?

**Comment by eteran** [comment]
Yea, since we use native/Qt file dialogs the special cases outlines for the custom nedit ones should be trimmed out.

Honestly, much of the early segments of the help could be rewritten/reworked as it was written before GUI text editors were super common on UNIX and users today (unlike when nedit first came out) can be expected to have a basic understanding of mouse/keyboard interaction for GUI programs.

For now, we'll just trim out sections that blatantly no longer apply.
